config/jest/jest-preset.json

```diff
@@ -0,0 +1,13 @@
+{
+  "automock": false,
+  "moduleNameMapper": {
+    "^[./a-zA-Z0-9$_-]+\\.(png|svg|jpg)$": "<rootDir>/node_modules/react-scripts/config/jest/ImageStub.js",
```

Comment:
This list would need to be more exhaustive: `jpg|png|gif|eot|svg|ttf|woff|woff2|mp4|webm`.